commands/update/check_update.go

```diff
@@ -43,7 +43,7 @@ func CheckUpdate(s *iostreams.IOStreams, version string, silentErr bool) error {
 			return nil
 		}
 		fmt.Fprintf(s.StdOut, "%v %v", c.GreenCheck(),
-			c.Green("You are already using the latest version of glab"))
+			c.Green("You are already using the latest version of glab\n"))
```

Comment:
Or maybe we could add the line break to the second param
```suggestion
		fmt.Fprintf(s.StdOut, "%v %v\n", c.GreenCheck(),
			c.Green("You are already using the latest version of glab"))
```

Reply:
I can totally live with that.